plugins/processors/scale/README.md

```diff
@@ -0,0 +1,76 @@
+# Scale Processor Plugin
+
+The scale processor filters for a set of fields,
+and scales the respective values from an input range into
+the given output range according to this formula:
+
+```math
+\text{result}=(\text{value}-\text{input\_minimum})\cdot\frac{(\text{output\_maximum}-\text{output\_minimum})}
+{(\text{input\_maximum}-\text{input\_minimum})} +
+\text{output\_minimum}
+```
```

Comment:
I realize others asked for this, but I really do not like this. This only is easily readable when looking at the readme on github where it is rendered.

No need to change, but I will follow-up with sven on this when he is back.

Reply:
Some IDE’s also support this. If not, via a plugin.